plugins/coverbrowser.koplugin/main.lua

```diff
@@ -330,36 +478,52 @@ function CoverBrowser:addToMainMenu(menu_items)
                     },
                     {
                         text = _("Show number of pages left to read"),
-                        enabled_func = function() return not BookInfoManager:getSetting("hide_page_info") end,
+                        enabled_func = function()
+                            return not BookInfoManager:getSetting("hide_page_info") and fc.display_mode_type == "list"
+                        end,
                         checked_func = function() return BookInfoManager:getSetting("show_pages_left_in_progress") end,
                         callback = function()
                             BookInfoManager:toggleSetting("show_pages_left_in_progress")
                             fc:updateItems(1, true)
                         end,
                     },
+                    {
+                        text = _("Show file metadata"),
```

Comment:
I don't think file extension and size are "metadata".

Reply:
Agreed. These are file properties, not metadata. (You might argue that properties are a specific subset of metadata in which case… yes, that's the point. :-)

Reply:
I wasn't going to argue that, happy to change it, in fact I thought I had changed it back to the original as at some point had used a way worse one, alas I didn't.